### ISSUE TITLE
gReview trying to access gerrit DB

### ISSUE THREAD

**Issue by galberti** [onepremise/gReview#32]
I see in the logs lots of 
ERROR [ajp-bio-127.0.0.1-8085-exec-1948] [GerritRepositoryAdapter:714] ALERT: <username> does not have "Access Database" capability.
but everything seems to work anyway. Besides maybe logging a warning, isn't using the gerrit REST interface more appropriate?

**Comment by onepremise** [comment]
That's a relatively benign error. Yes, it probably should be a just a warning. Basically, the user you're using to connect with doesn't have "Access Database" granted under Global Capabilities in Gerrit. You can still use the addon, it just won't pull as much user data for display purposes. General functionality should work. Otherwise, if you have permissions, you can grant your user the "Access Database" priv in Gerrit.

**Comment by galberti** [comment]
Yes, everything works anyway, that's why I thought a warning would have been better.. what about using the gerrit's REST interface to get those information? AFAIK that doesn't need any special permission on the DB.
